app/controllers/api/todos_controller.rb

```diff
@@ -1,4 +1,4 @@
-class Api::V1::TodosController < ApplicationController
+class Api::TodosController < ApplicationController
```

Comment:
今回のPRでは大丈夫ですが, controllerにフォルダーが多くなるようでしたらmoduleにした方がよさそうですね！
```suggestion
module Api
  class TodosController < ApplicationController
```

参考までに